BootloaderCorePkg/Tools/CfgDataTool.py

```diff
@@ -21,7 +21,8 @@ class CDATA_BLOB_HEADER(Structure):
         ('Signature', ARRAY(c_char, 4)),
         ('HeaderLength', c_uint8),
         ('Attribute', c_uint8),
-        ('Reserved', ARRAY(c_char, 2)),
+        ('Svn', c_uint8),

```

Comment:
Let us remove the full class CDATA_BLOB_HEADER to avoid duplicated definitions.
We can always use CCfgData.CDATA_BLOB_HEADER instead.

Reply:
Done.